MvvmCross/Droid/Droid/Views/MvxAndroidViewPresenter.cs

```diff
@@ -662,9 +655,30 @@ protected virtual string FragmentJavaName(Type fragmentType)
         protected virtual Fragment GetFragmentByViewType(Type type)
         {
             var fragmentName = FragmentJavaName(type);
-            var fragment = CurrentFragmentManager.FindFragmentByTag(fragmentName);
+            return CurrentFragmentManager.FindFragmentByTag(fragmentName);
+        }
+
+        protected virtual MvxBasePresentationAttribute GetOverridePresentationAttribute(Type viewType)
+        {
+            if (viewType?.GetInterface(nameof(IMvxOverridePresentationAttribute)) != null)
+            {
+                var viewInstance = Activator.CreateInstance(viewType) as IMvxOverridePresentationAttribute;
```

Comment:
Do we explicitly need to dispose this after using it?

Reply:
It does seem like it will be a good idea. From the [Xamarin docs](https://developer.xamarin.com/guides/android/advanced_topics/garbage_collection/#Object_Cycles):
>To shorten object lifetime, Java.Lang.Object.Dispose() should be invoked. This will manually "sever" the connection on the object between the two VMs by freeing the global reference, thus allowing the objects to be collected faster."

I have push an update wrapping the viewInstance in a using. Allowing it to get dispose of once we are done with it.